### PR TITLE
Add agnostic link to help

### PR DESCRIPTION
**Pull request by ponceta**
Contributes to #2238

Does not work for RM since https://help.geo.admin.ch/?lang=rm does not exist

@oterral / @gjn Do I create a new translation for that or do you have an angular magic trick for this?

like if `rm` redirect to `de`.

### CODE REVIEW

**Comment by ponceta** [comment]
I'll remove directly the tokens once redirection of rm wil be fixed.

File: src/index.mako.html [hunk 1]
```diff
@@ -173,7 +173,7 @@
           <a href="" ng-click="globals.feedbackPopupShown = !globals.feedbackPopupShown">
             <span ng-class="{'selected': globals.feedbackPopupShown}" translate>problem_announcement</span>
           </a>&nbsp;&nbsp;
-          <a target="_blank" ng-href="{{topicId + '_help_link_href' | translate}}" translate>help_label</a>&nbsp;&nbsp;
+          <a target="_blank" ng-href="//help.geo.admin.ch/?lang={{langId}}" translate>help_label</a>&nbsp;&nbsp;
```
Comment (oterral) [inline src/index.mako.html]: I'm not 100% sure but I think we don't need to use `ng-href`

Comment (ponceta) [inline src/index.mako.html]: Really? But if you use a std href, {{langId}} won't be interpreted correctly! No?

Comment (oterral) [inline src/index.mako.html]: ng-href allow two way data binding (read/write) in our case we need just read.

Comment (ponceta) [inline src/index.mako.html]: You're right :+1:

**Comment by ponceta** [comment]
search bern failed

jenkins test

**Comment by oterral** [comment]
jenkins test

**Comment by ponceta** [comment]
jenkins test

**Comment by gjn** [comment]
Very nice. Thanks alot.

**Pull request merged**